tests/org.jboss.reddeer.jface.test/src/org/jboss/reddeer/jface/test/viewer/PrepareTreeWithStyledItems.java

```diff
@@ -68,16 +68,16 @@ public void run() {
 
 	@After
 	public void cleanup() {
-		org.jboss.reddeer.core.util.Display.syncExec(new Runnable() {
+		org.jboss.reddeer.common.util.Display.syncExec(new Runnable() {
 			@Override
 			public void run() {
-				for (Shell shell : org.jboss.reddeer.core.util.Display.getDisplay().getShells()) {
+				for (Shell shell : org.jboss.reddeer.common.util.Display.getDisplay().getShells()) {
 					if (shell.getText().equals(title)) {
 						shell.dispose();
 						break;
 					}
 				}
-				new WaitWhile(new ShellWithTextIsActive(title));
+				new WaitWhile(new ShellWithTextIsAvailable(title));
```

Comment:
you have here waiting inside UI thread. It was done this way already but we can fix it, could you look at it?